<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewerClass</class>
 <widget class="QMainWindow" name="ImageViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>873</width>
     <height>651</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Controls</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonReset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFSHS">
         <property name="text">
          <string>FSHS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEH">
         <property name="text">
          <string>EH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLinCon">
         <property name="text">
          <string>LinCon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEdges">
         <property name="text">
          <string>Edges</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>LD/RT</string>
         </property>
         <widget class="QPushButton" name="pushButtonRVT">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>110</y>
            <width>81</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>LD</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditTau">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>41</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBoxT">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>50</y>
            <width>51</width>
            <height>26</height>
           </rect>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>63</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Tau:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>21</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>T:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>16</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>K:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditK">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>80</y>
            <width>61</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>100</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonRT">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>150</y>
            <width>91</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Rouy-Tourin</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>PM/G_MCF</string>
         </property>
         <widget class="QPushButton" name="pushButtonPM">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>100</y>
            <width>71</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>PM</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>sigma:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditSigma">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>41</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>0.5</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonGMCF">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>160</y>
            <width>71</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>GMCF</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonMCF">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>130</y>
            <width>71</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>MCF</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>60</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Epsilon:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditEpsilon">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>60</y>
            <width>31</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>0.1</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>725</width>
         <height>627</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>scrollArea</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionInvert"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="text">
    <string>Resize</string>
   </property>
  </action>
  <action name="actionInvert">
   <property name="text">
    <string>Invert</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImageViewer.qrc"/>
 </resources>
 <connections/>
</ui>
